gw_eccentricity/eccDefinition.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
             - "hlm"
             - "amplm" and "phaselm"
 
-            Note that providing "hlm", "amplm" and "phaselm" all at the same
+            Note that providing "hlm" and "amplm" or "phaselm" at the same
```

Comment:
Very confusing, make the whole thing simpler:
and one of the followings are mandatory:
 - "hlm" OR
 - "amplm" and "phaselm"
 but not both at the same time.

Reply:
where I also did "one of the followings are" --> "one of the following is"

Reply:
I edited my suggestion, please take a look now.

Reply:
Done